traveler-ios-ui/TravelerKitUI/ViewControllers/OrderSummaryViewController.swift

```diff
@@ -133,6 +133,24 @@ class OrderSummaryViewController: UITableViewController {
 
         tableView.deselectRow(at: indexPath, animated: true)
     }
+
+    override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        let cellIdentifier = tableView.cellForRow(at: indexPath)?.reuseIdentifier
+
+        if cellIdentifier == infoCellIdentifier {
+            return true
+        } else {
+            return false
+        }
```

Comment:
This is correct

A better way (more extensible way) to write this would be

```
switch tableView.cellForRow(at: indexPath)?.reuseIdentifier {
case infoCellIdentifier?:
  return true
default:
  return false
}
```